epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyVisibilityItem.java

```diff
@@ -153,6 +164,13 @@ private boolean isInFocusVisible() {
         : totalArea == visibleArea;
   }
 
+  private boolean isPartiallyVisible(int threshold) {
+    final int totalArea = height * width;
+    final int visibleArea = visibleHeight * visibleWidth;
+
+    return (visibleArea / (float)totalArea) * 100 >= threshold;
```

Comment:
Looks like we should not send event if `threshold` is 0 (default value) ?

Reply:
good point

Reply:
832f83d3fbb076502d94184d6361b722d2966bab